### PR TITLE
Add rule for self requested reviews

### PR DESCRIPTION
**Pull request by tupaschoal**

### CODE REVIEW

File: CommunityGuidelines.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ On top of the community rules, the development and moderation team has to keep t
 - Any contributor PR requires at least two code reviews, either by developers or trusted community members (at least one developer).
 - Any developer PR requires at least one code review, by another developer.
 - Any PR requires one playtest review, if applicable.
+- Any developer may self-request a review on a PR. This will expire 1 day after the PR gets another developer review.
```
Comment (Gymnasiast) [inline CommunityGuidelines.md:35]: This is not what we discussed. We were talking about 2 days after notification of the developer who self-requested.

**Review by Gymnasiast — CHANGES_REQUESTED** [review]

File: CommunityGuidelines.md (hunk 1 above)
Comment (tupaschoal) [inline CommunityGuidelines.md:35]:
```suggestion
- Any developer may self-request a review on a PR. It expires 2 days after being notified that the PR is ready to merge as it already has another developer's approval.
```

File: CommunityGuidelines.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ On top of the community rules, the development and moderation team has to keep t
 - Any contributor PR requires at least two code reviews, either by developers or trusted community members (at least one developer).
 - Any developer PR requires at least one code review, by another developer.
 - Any PR requires one playtest review, if applicable.
+- Any developer may self-request a review on a PR. It expires 2 days after being notified that the PR is ready to merge as it already has another developer's approval.
```
Comment (Gymnasiast) [inline CommunityGuidelines.md:35]: This is good, but I think this makes it clearer:

```suggestion
- Any developer may self-request a review on a PR. If another developer approves the PR, the developer who self-requested a review should be notified via Discord. If the developer does not leave a review within two days after this notification, the PR may be merged.
```

**Review by Gymnasiast — APPROVED** [review]

**Review by karst — APPROVED** [review]
LGTM

**Pull request merged**